specification/UAVCAN_Specification.tex

```diff
@@ -51,14 +51,41 @@ \section*{Support and feedback}
 Information, documentation, and discussions related to UAVCAN are available via the official website at
 \href{http://uavcan.org}{uavcan.org}.
 
-\section*{Legal statement}
+\section*{License}
 
-UAVCAN is an interface standard open to everyone.
+UAVCAN is a standard open to everyone.
 No licensing or approval of any kind is necessary for its implementation, distribution, or use.
 
-In no event shall the authors of the standard be liable for any damage arising, directly or indirectly, from its use.
+% The following statement looks a bit archaic, but it is the recommended form according to
+% https://creativecommons.org/choose/results-one?license_code=by&amp;jurisdiction=&amp;version=4.0&amp;lang=en
+This work is licensed under the Creative Commons Attribution 4.0 International License.
+To view a copy of this license, visit \url{http://creativecommons.org/licenses/by/4.0/}
+or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
+
+\begin{center}
+    \includegraphics[width=0.2\textwidth]{cc-by}
+\end{center}
+
+\BeginRightColumn
+
+\section*{Disclaimer of warranty}
+
+NOTE WELL: This Specification is provided on an ``AS IS'' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+express or implied, including, without limitation, any warranties or conditions of
+TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE.
+
+\BeginRightColumn
+
+\section*{Limitation of liability}
 
-This UAVCAN specification is distributed under the terms of the Creative Commons Attribution 4.0 International License.
+In no event and under no legal theory, whether in tort (including negligence), contract, or otherwise,
+unless required by applicable law (such as deliberate and grossly negligent acts) or agreed to in writing,
+shall any author of this Specification be liable for damages,
+including any direct, indirect, special, incidental, or consequential damages of any character arising
+from, out of, or in connection with the Specification or the implementation, deployment,
+or other use of the Specification (including but not limited to damages for loss of goodwill,
+work stoppage, computer failure or malfunction, or any and all other commercial damages or losses),
```

Comment:
> (including but not limited to damages for loss of goodwill,
work stoppage, computer failure or malfunction, or any and all other commercial damages or losses)

Perhaps we should make this example more domain specific. My layman's guess is that in litigation such examples serve to strengthen the "we couldn't have been more clear" argument. So how about:

(including but not limited to damages for loss of goodwill,
work stoppage, equipment failure or malfunction, injuries to persons, or any and all other commercial damages or losses)